app.js

```diff
@@ -3,6 +3,7 @@ var cookieParser = require('cookie-parser')
 var bodyParser = require('body-parser')
 
 const orders = require('./routes/orders')
+const toppings = require('./routes/toppings')
```

Comment:
I'd use `import` rather than `require()`.

Reply:
Not to belabor the process, but is this suggestion due to convention or effectiveness?

Reply:
Under the hood, there is a difference. `require()` is commonJS modules, `import` is ES6. Both accomplish the same thing, but I prefer the ES6 way of doing things, and the standard is a bit more developed. Sometimes you have to use one over the other if the library in question only allows one of the two, but in this case either work, it's a matter of preference. A bit more info can be found here: https://stackoverflow.com/questions/31354559/using-node-js-require-vs-es6-import-export

Reply:
Yeah, in general I'd go with `import` everywhere. It's groovier, if nothing else.